openshift2nulecule/openshift.py

```diff
@@ -162,17 +163,20 @@ def __init__(self, artifacts):
 
         # get all images of all ReplicationControllers
         self.images = []
-        for artifact in self.artifacts:
-            # TODO: add support for other kinds (Pod, ....?)
-            if artifact["kind"] in ["ReplicationController", "DeploymentConfig"]:
-                self.images.extend(utils.get_image_info(artifact))
+        for provider in NULECULE_PROVIDERS:
+            for artifact in self.artifacts[provider]:
+                # TODO: add support for other kinds (Pod, ....?)
+                if artifact["kind"] in ["ReplicationController",
+                                        "DeploymentConfig"]:
+                    self.images.extend(utils.get_image_info(artifact))
 
     def _remove_securityContext(self):
         """
-        Remove securityContext from all objects in kind_list.
+        Remove securityContext from all objects in kind_list
+        for Kubernetes artifacts.
         """
 
-        for obj in self.artifacts:
+        for obj in self.artifacts["kubernetes"]:
```

Comment:
You are of course right we don't need this fce now when we are not exporting `pods`.
But we might do it in future again. and I hope that we will do.

But this PR is not really about this.